types/src/get_with_proof.rs

```diff
@@ -197,7 +197,22 @@ pub fn verify_update_to_latest_ledger_response<Sig: Signature>(
 
     // Verify ledger info signatures.
     if !(ledger_info.version() == 0 && signatures.is_empty()) {
-        validator_verifier.batch_verify_aggregated_signature(ledger_info.hash(), signatures)?;
+        // Verify epoch changes
+        if ledger_info.epoch() > epoch_info.epoch {
```

Comment:
Will `ledger_info.epoch()` always be 1 greater than `epoch_info.epoch`, or can it be (e.g.) 10 greater? I'm wondering about the asymmetry of the `>` here and the `+ 1` below.

Reply:
sorry for the bad naming, it's not `epoch_info.epoch += 1`, it's `epoch_li.epoch() + 1` which we can assert equal to `ledger_info.epoch()`